experiments/compositions/composition/tests/data/TestSimpleAddFacadeField/input.yaml

```diff
@@ -99,6 +96,32 @@ subjects:
   name: composition-controller-manager
   namespace: composition-system
 ---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
Do we need this to work ?

Reply:
wouldnt * * * in the actual rbac be sufficient. Why do we need this ?

Reply:
I believe at some point the plan is to remove the  * * * from the cluster role, at which point this will be necessary. I made all the changes without the superuser on the cluster role first, so I know all the tests will work as-is once that workaround is gone.